unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/DCLCharacterController.cs

```diff
@@ -102,6 +102,10 @@ public class DCLCharacterController : MonoBehaviour
     [System.NonSerialized]
     public float movingPlatformSpeed;
 
+    [HideInInspector] public System.Action OnJump;
+    [HideInInspector] public System.Action OnHitGround;
+    [HideInInspector] public System.Action<float> OnMoved;
```

Comment:
The `[HideInInspector]` attribute is not needed here, as the System.Action is not serializable. Also, we should add the `event` modifier, like this:

```
public event System.Action OnJump
```
This modifier prevents other classes from invoking the event, making the code better encapsulated.